docs/src/installation/Raspberry-Pi-64.md

```diff
@@ -0,0 +1,84 @@
+# 64 bit Raspery OS build guide
+
+To run spotifyd on a 64 bit Raspberry Pi OS, you have two possiblities. Compile the 64 bit binary by yourself or add the 32 bit architecture as 
+an additional architecture to your 64 bit Raspberry Pi OS.
+
+Let's start with compiling the 64 bit version, as it offers you more possibilities like adding the DBus feature for controlling the service with
+DBus commands.
+
+## Option 1: Compiling the 64 bit version
```

Comment:
Ideally, this could replace the current `Compiling from source (Ubuntu)` section in the current book, since these instructions apply to most distributions. But we can also leave it here, if you prefer.

Reply:
Also fine, you could rename it simply to "Compiling from source" and leave the heading and a link here. :)
I noticed, that Ubuntu hasn't any dev packages preinstalled, so I would add something:

---
Depending on the distribution, development packages need to be installed. A complete set e.g. for Ubunto which comes without develepment packages preinstalled woudl be:

`sudo apt install curl git build-essential pkg-config libasound2-dev`

For certein build flags, more packages may be required, like the dbus_mpris flag:
`sudo apt install libdbus-1-dev`
---

I have not tested if build without dbus_mpris flag doesn't required the libdbus-1-dev package, but I guess so. Maybe I can test on the weekend.